Lib/ufo2ft/postProcessor.py

```diff
@@ -379,3 +389,11 @@ def _stripCharStringWidth(program):
     if stack:
         result.extend(stack)
     return result
+
+
+def _reloadFont(font: TTFont) -> TTFont:
+    """Recompile a font to arrive at the final internal layout."""
+    stream = BytesIO()
+    font.save(stream)
+    stream.seek(0)
+    return TTFont(stream)
```

Comment:
Would passing `lazy=True` to `TTFont` constructor make any measurable difference here?

Reply:
I can measure no appreciable wall time improvement but have not profiled in depth.

Reply:
`lazy=True` doesn't mix well with varLib merger. Don't use for now.